azure-pipelines.yml

```diff
@@ -1,207 +0,0 @@
-trigger:
```

Comment:
i'm pretty sure you can't delete this file

Reply:
🤦  I was not meant to delete this. Thanks for spotting it